src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -663,8 +663,15 @@ internal virtual void PrepareSession()
             {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-macos`
Web cmdlets tests using the cmdlet's aliases.Web cmdlets ignore headers with null value
```powershell
A parameter cannot be found that matches parameter name 'Not-Throw'.
at <ScriptBlock>, /Users/runner/runners/2.163.1/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1: line 3490
3490:         { Invoke-WebRequest -Uri $uri -ContentType $null } | Should -Not-Throw

```